packages/core/admin/admin/src/content-manager/components/DynamicTable/index.js

```diff
@@ -106,9 +107,10 @@ const DynamicTable = ({
           showPublish={canPublish && hasDraftAndPublish}
           showDelete={canDelete}
           onConfirmDeleteAll={onConfirmDeleteAll}
+          onConfirmPublishAll={onConfirmPublishAll}
+          onConfirmUnpublishAll={onConfirmUnpublishAll}
           selectedEntries={selectedEntries}
           clearSelectedEntries={clearSelectedEntries}
-          handleBulkPublish={handleBulkPublish}
         />
```

Comment:
@markkaylor I've been naming everything to match the pre-existing `onConfirmDeleteAll`, but now I'm thinking I could rename everything to match your format: `handleBulkPublish`, `handleBulkUnpublish`, `handleBulkDelete` if you prefer

Reply:
Thanks for bringing it up. I think what you have is better. I already updated my [new PR ](https://github.com/strapi/strapi/pull/16559/files/61c168f7442ca36d93ca5d95a8f9cd79ad430e30) to match yours, I meant to do it before I merged the validations pr.

Reply:
the nice thing about handleBulkPublish is that it kind of matches the name of the backend controller

Reply:
I'll have to stare at it some more 🤣 . I can make the update on my end though after yours is merged if we decide that's better.